application/src/Api/Adapter/AdapterInterface.php

```diff
@@ -1,6 +1,7 @@
 <?php
 namespace Omeka\Api\Adapter;
 
+use Omeka\Api\Representation\RepresentationInterface;
```

Comment:
This is just for its appearance in the docblock for getRepresentation, yeah? Would prefer to fix these by fully-qualifying within the docblock rather than a `use`.

Reply:
Ok. Fixed in next commit.